site/gatsby-site/src/components/discover/SORTING_LISTS.js

```diff
@@ -4,6 +4,7 @@ const SORTING_LIST = [
   {
     default: true,
     name: 'relevance',
+    value: 'instant_search-en',
```

Comment:
By default, it should be `en`. Why do you need this?

Reply:
The Algolia [sortBy widget](https://www.algolia.com/doc/api-reference/widgets/sort-by/js/) needs its items to have a value for its first iteration. Since that wasn't existing it was returning the "undefined" warning.

Reply:
Yeah, but language is never empty, on the first render it already knows the language, it should go directly to `value_en`. I think there is some executive order that needs to be tweaked 🤔

Reply:
Yeah that's what I thought also, but after debugging I realized the issue happens when the Sorting Algolia widget is mounted, and it uses the SORTING_LIST as its `items`. So the first time, when it mounts, it automatically tries to find the `value` field that doesn't exist. I'll dig more into this next week but I couldn't find another way to fix this.